doc/v2/documentation_data_format.rst

```diff
@@ -1417,7 +1417,7 @@ The following functions are supported:
      - float
    * - ``sign(x)``
      - | sign of ``x``
-       | ``piecewise(1, x > 0, -1, x < 0, 0)``
+       | ``piecewise(1, x>=0, -1)``
```

Comment:
Wikipedia says otherwise, and I don't see a sign function in SBML. What is this based on now?

https://en.wikipedia.org/wiki/Sign_function#Definition

Reply:
You are right. No intended change, I just regenerated the RST tables. It seems this was fixed in the wrong place last time...